src/host/getset.cpp

```diff
@@ -859,7 +859,9 @@ void ApiRoutines::GetLargestConsoleWindowSizeImpl(const SCREEN_INFORMATION& cont
             // GH#3490 - If we're in conpty mode, don't invalidate the entire
             // viewport. In conpty mode, the VtEngine will later decide what
             // part of the buffer actually needs to be re-sent to the terminal.
-            if (!(g.getConsoleInformation().IsInVtIoMode() && g.getConsoleInformation().GetVtIo()->IsResizeQuirkEnabled()))
+            const auto inConpty{ g.getConsoleInformation().IsInVtIoMode() };
+            const auto resizeQuirk{ g.getConsoleInformation().GetVtIo()->IsResizeQuirkEnabled() };
+            if (!(inConpty && resizeQuirk))

```

Comment:
FWIW this change is not quite the same as it was before as it prevents the compiler from shortcutting the latter call. An alternative way to write this may be this:
```cpp
if (!g.getConsoleInformation().IsInVtIoMode() ||
    !g.getConsoleInformation().GetVtIo()->IsResizeQuirkEnabled())
{
```

Reply:
ah sorry, this was for debugging.